tests/fsharpqa/Source/Warnings/WarnIfPossibleDotNetPropertySetter.fs

```diff
@@ -0,0 +1,13 @@
+// #Warnings
+//<Expects status="Warning" span="(10,5)" id="FS0020">If you intended to set a value to a property, then use the '<-' operator e.g. 'x <- expression'</Expects>
```

Comment:
Shouldn't this say `e.g. Obj.Prop <- expression` instead of `e.g. x <- expression`?